### DISCUSSION TITLE
Suggestion to open Track&Trace decision log and add transparency in prioritization in Project-Origin

### DISCUSSION THREAD

**Discussion by AndreAlnor** [project-origin/.github#24] [General]
Hi everyone

Just left Energinet to set up shop leveraging on this project to create hosted services for granular certification. Landing on the outside of the Energinet, 50Hz, Elering collaboration it dawned on me how difficult it is to follow or participate in the market and technical discussions. Without this input I find it difficult to follow or help set the prioritization for this project.

I suggest to make a formal request to the Energy Track & Trace project to open their decision log for collaboration and/or create a log or discussion in this project where we can debate, prioritize and log decisions. If it can help I would be happy to host a 30 minuter bi weekly open collaborator call to facilitate.

I will be happy to do the work this requires. Before I do this I would however like to hear you guys opinion on this?

Looking forward to continue the good work in this venue with you :)

Br
André

**Comment by MartinSchmidt** [comment]
Hi @AndreAlnor I think it would be a great ideer to begin a open track for ProjectOrigin, you can use the announcements to publish a teams meeting where people can attend.
